src/xref2/link.ml

```diff
@@ -13,9 +13,10 @@ let locations env id locs =
 (** Equivalent to {!Comment.synopsis}. *)
 let synopsis_from_comment (docs : Component.CComment.docs) =
   match docs with
-  | ({ value = #Comment.nestable_block_element; _ } as e) :: _ ->
-      (* Only the first element is considered. *)
-      Comment.synopsis [ e ]
+  | { value = `Paragraph (_, text); _ } :: _ -> Some text
+  (* | ({ value = #Comment.nestable_block_element; _ } as e) :: _ -> *)
```

Comment:
What's happening here?

Reply:
Before, the `Component` version of nestable block comments were identical to the `Lang` version. Now, they differ, so we cannot anymore call the `Lang` function.

The old code now commented slipped in my staged area, I will clean it.